tableauserverclient/models/workbook_item.py

```diff
@@ -275,6 +276,10 @@ def from_response(cls, resp, ns):
                 data_acceleration_config,
             ) = cls._parse_element(workbook_xml, ns)
 
+            # workaround for Personal Space workbooks which won't have a project
+            if not project_id:
```

Comment:
So this just inserts a meaningless project id to avoid code crashing on the empty value, is that right? Is it possible to put in a human-readable string that makes this more obvious, instead of a uuid?

Reply:
Yes that's right. And upon further thought this is not a great way to solve it, especially because it will populate project_id with something that _looks like_ a project ID but really isn't.